src/pages/Lend/LoanSearchPage.vue

```diff
@@ -354,6 +371,14 @@ export default {
 				}
 			}
 		}
+
+		const data = this.apollo.readQuery({
+			query: basketCountQuery,
```

Comment:
Is see in this changeset that we're pulling from the Apollo cache here. Where is this query being ran originally?